cns/networkcontainers/networkcontainers_linux.go

```diff
@@ -3,20 +3,180 @@
 
 package networkcontainers
 
-import "github.com/Azure/azure-container-networking/cns"
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"go/types"
+	"io/ioutil"
+	"os"
+	"os/exec"
+
+	"github.com/Azure/azure-container-networking/cns"
+	"github.com/Azure/azure-container-networking/log"
+	"github.com/containernetworking/cni/libcni"
+	"github.com/containernetworking/cni/pkg/invoke"
+)
+
+const (
+	VersionStr string = "cniVersion"
+	PluginsStr string = "plugins"
+	NameStr    string = "name"
+)
 
 func createOrUpdateInterface(createNetworkContainerRequest cns.CreateNetworkContainerRequest) error {
-	return nil
+
+	if createNetworkContainerRequest.NetworkContainerType == cns.WebApps {
+		log.Printf("[Azure CNS] Operation not supported for WebApps Orchestrator.")
+		return nil
+	}
+
+	exists, _ := interfaceExists(createNetworkContainerRequest.NetworkContainerid)
+	if !exists {
+		log.Printf("[Azure CNS] Only Update Operation is supported.")
+		return nil
+	}
+
+	return createOrUpdateWithOperation(createNetworkContainerRequest, "UPDATE")
 }
 
 func setWeakHostOnInterface(ipAddress string) error {
 	return nil
 }
 
 func createOrUpdateWithOperation(createNetworkContainerRequest cns.CreateNetworkContainerRequest, operation string) error {
+	log.Printf("[Azure CNS] createOrUpdateWithOperation called with operation type %v", operation)
+
+	if _, err := os.Stat("/opt/cni/bin/azure-vnet"); err != nil {
```

Comment:
introduce constants for these paths

Reply:
you can also set them as default values in main since we will take args for user to set CNI path and conflist path